README.md

```diff
@@ -127,6 +131,7 @@ PGPASSWORD=test-password pg_restore -h postgre-db -U postgres -d pragma /backup.
 ### 3. Export the required environment variables:
 
 ```bash
+export MODE=dev
```

Comment:
probably `DEV_MODE=1` is more explicit

Reply:
I like having `Mode` as an enum as it will allow us to allow more modes later by extending it (`staging`, `test` ...). what do you think? maybe a YAGNI case, no strong opinions